package.json

```diff
@@ -170,9 +170,19 @@
             "default": false
           },
           "calva.enableClojureLspOnStart": {
```

Comment:
Does changing this setting require a reload of the window for it to take effect? If so I think we should state that in the description.